### PR TITLE
Add network ID to container inspect

### PR DESCRIPTION
**Pull request by wenchma**
Fixes issue #19089

Signed-off-by: Wen Cheng Ma wenchma@cn.ibm.com

### CODE REVIEW

File: docs/reference/api/docker_remote_api_v1.22.md [hunk 1]
```diff
@@ -542,14 +546,15 @@ Return low-level information on the container `id`
...
-					"MacAddress": ""
+					"NetworkID": "7ea29fc1412292a2d7bba362f9253545fecdfa8ce9a6e37dd10ba8bee7129812",
+          "EndpointID": "7587b82f0dada3656fda26588aee72630c6fab1536d36e394b2bfbcf898c971d",
+          "Gateway": "172.17.0.1",
+          "IPAddress": "172.17.0.2",
+          "IPPrefixLen": 16,
+          "IPv6Gateway": "",
+          "GlobalIPv6Address": "",
+          "GlobalIPv6PrefixLen": 0,
+          "MacAddress": "02:42:ac:12:00:02"
```
Comment (thaJeztah) [inline docs/reference/api/docker_remote_api_v1.22.md]: Can you fix the indentation here?

File: container/container_unix.go [hunk 2]
```diff
@@ -523,7 +524,7 @@ func (container *Container) UnmountIpcMounts(unmount func(pth string) error) {
 			warnings = append(warnings, err.Error())
 		} else if shmPath != "" {
 			if err := unmount(shmPath); err != nil {
-				warnings = append(warnings, fmt.Sprintf("failed to umount %s: %v", shmPath, err))
+				warnings = append(warnings, fmt.Sprintf("failed to unmount %s: %v", shmPath, err))
```
Comment (thaJeztah) [inline container/container_unix.go]: I wonder if this was on purpose (and referring to [`umount`](http://man7.org/linux/man-pages/man8/umount.8.html)), either way, I think it's fine

**Comment by thaJeztah** [comment]
Adding this to the milestone, not sure if "design review" is actually needed, so feel free to skip forward to "code review"

/cc @mrjana

File: docs/reference/api/docker_remote_api_v1.22.md (hunk 1 above)
Comment (wenchma) [inline docs/reference/api/docker_remote_api_v1.22.md]: will fix it

File: container/container_unix.go (hunk 2 above)
Comment (wenchma) [inline container/container_unix.go]: @thaJeztah maybe you are right :smile:

**Comment by mavenugo** [comment]
LGTM.

Just a note (I dont think we must document it... but wont harm I guess) :
the network-id for default bridge network will change across daemon restarts.
That's because default-bridge network is the only network for which the network
configuration can change across reboots (due to the legacy daemon configurations).

**Comment by tiborvass** [comment]
LGTM

**Comment by thaJeztah** [comment]
docs LGTM

ping @vdemeester @MHBauer for review

**Comment by vdemeester** [comment]
docs LGTM 🐸

**Comment by calavera** [comment]
Userns build completed successfully but GitHub was never notified:
```
14:59:05 ---
14:59:05 
14:59:05 Error response from daemon: no such id: docker-pr-userns4914
14:59:05 Error: failed to remove containers: [docker-pr-userns4914]
14:59:05 Notifying endpoint 'HTTP:https://leeroy.dockerproject.org/notification/jenkins'
14:59:06 Finished: SUCCESS
```

Merging! :tada:

**Pull request merged**